### DISCUSSION TITLE
No audio in video recordings

### DISCUSSION THREAD

**Discussion by Teklock** [blakeblackshear/frigate#6789] [General]
I just noticed that my recordings don't have audio.  Is audio normally recorded with video in a standard h.264/AAC (rtsp://*:*@192.168.1.120:554/cam/realmonitor?channel=1&subtype=0) Dahua rtsp feed?  This stream is only for recordings with no detection.  Audio works fine when viewing live (rtc/mse) and I'm not seeing anything in the logs.  Thanks!

**Comment by NickM-27** [comment]
This is covered in the docs https://docs.frigate.video/troubleshooting/faqs#audio-in-recordings

**Reply by Teklock** [reply]
I can't believe I missed that with the number of times I've read the docs.  gezz  Thank you sir!
